language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/FunctionGenerator.java

```diff
@@ -52,7 +52,8 @@
  */
 public class FunctionGenerator {
 
-    public static final Pattern FULLY_QUALIFIED_MODULE_ID_PATTERN = Pattern.compile("([\\w]+)\\/([\\w.]+):(.+):");
+    public static final Pattern FULLY_QUALIFIED_MODULE_ID_PATTERN = Pattern.compile("([\\w]+)\\/([\\w.]+):([^:]+):" +
```

Comment:
Try to get the pattern to a single line, otherwise it is bit hard to read

Reply:
Updated.